cclib/parser/molcasparser.py

```diff
@@ -40,17 +40,29 @@ def normalisesym(self, label):
     def before_parsing(self):
         # Compile the regex for extracting the element symbol from the
         # atom label in the "Molecular structure info" block.
-        self.re_atomelement = re.compile('([a-zA-Z]+)\d+')
+        self.re_atomelement = re.compile('([a-zA-Z]+)\d?')
 
         # Compile the dashes-and-or-spaces-only regex.
         self.re_dashes_and_spaces = re.compile('^[\s-]+$')
 
+        # Molcas can do multiple calculations in one job, and each one
+        # starts from the gateway module. Onle parse the first.
+        # TODO: It would be best to parse each calcualtion as a separate
```

Comment:
`calcualtion -> calculation`

Reply:
Fixed in a0671265b22f7a1e8afb76d929405398f3fc1499